examples/typeorm-encryption/run-vault.sh

```diff
@@ -55,10 +59,8 @@ user PERSONS (
   ssn SSN COMMENT 'Social security number',
 )"
 
-# Install dependencies
-npm ci
+# Build
+yarn build
```

Comment:
Revert - The `start` script is using `ts-node` to run ts files directly so need to run `yarn build` but we do need to install dependencies so `npm ci` is needed.
```suggestion
# Install dependencies
npm ci
```

Reply:
it did work for me. The CI actually didn't but perhaps due to another problem